modules/ROOT/pages/_partials/data-sync/client-using-a-data-sync-client-to-query-a-mutation-on-a-data-sync-server.adoc

```diff
@@ -0,0 +1,101 @@
+[id="client-using-a-data-sync-client-to-query-a-mutation-on-a-data-sync-server-{context}"]
+= Using a Data Sync Client to query a mutation on a Data Sync Server
+
+Build a Data Sync Server and query the mutation.
+The Data Sync Server will be running locally in this procedure.
+
+.Prerequisites
+
+* The example in link:https://github.com/aerogear/mobile-docs/blob/master/modules/ROOT/pages/_partials/data-sync/server-create-a-graphql-schema-for-an-existing-data-source.adoc[Getting started with an example data source schema] runs locally.
+* Basic knowledge of `npm`.
+
+.Procedure
+
+. Create a basic link:https://webpack.js.org/guides/getting-started/[_webpack project_] repo.
++
+. Open a terminal and navigate into the cloned project.
++
+. To add the necessary Data Sync Client javascript libraries to your basic _webpack project_, execute the following commands:
```

Comment:
should it match https://mdc-docs.netlify.com/aerogear/latest/ds-query/ ?

Reply:
I think this is the next phase of this work as per your email where you outlined this structure: https://docs.google.com/spreadsheets/d/1eebcc4IGWnD-0xI4qYTly3pLyXxVujO17121WfOgf1c/edit?usp=sharing

I hope that we can use what is done in this PR to further the next phase :+1: